pages/_document.js

```diff
@@ -37,6 +37,7 @@ export default class MyDocument extends Document {
           <link rel="stylesheet" href="/static/style.css" />
           {this.props.styleTags}
         </Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
```suggestion
```

Reply:
In addition to the point mentioned above, adjust the pr description, as the task link in jira is divergent with the task performed in pr